run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+SRC="/app/extra/wt.tar.gz"
+
+# Extract
+if [[ ! -f "$XDG_DATA_HOME/sha_check" ]] || [[ "$(sha256sum /app/extra/wt.tar.gz | cut -d' ' -f1)" != "$(cat "$XDG_DATA_HOME"/sha_check 2>/dev/null)" ]]; then
+	sha256sum /app/extra/wt.tar.gz | cut -d' ' -f1 > "$XDG_DATA_HOME"/sha_check
+	tar -xv --gzip -f "$SRC" -C "$XDG_DATA_HOME"
```

Comment:
Is it because the launcher updates itself?

Reply:
No. It's because the launcher is located at `$XDG_DATA_HOME` due to it being coded by the game developers. We can't have it in /app/extra because the launcher (i.e. executable) lives in the same directory as the configuration files, so we were forced to use this workaround.